docs/plugins/plugin-development-guide/summary.rst

```diff
@@ -0,0 +1,34 @@
+Summary
+-------
+
+Congratulations! You've created your first, fully tested and documented, customization to Sylius inside a Sylius plugin!
+As you can see, there are some things to do at the beginning of development, but know, when you are already familiar with
+the whole structure, each next feature can be provided faster than the previous one.
+
+As a result, you should see a new field in product variant form:
+
+.. image:: ../../_images/product_variant_available_on_demand.png
+    :align: center
+
+What's next?
```

Comment:
I'd mention dropping us an email about the new plugin to contact@sylius.com. (for now) ;)